clients/java/revapi.json

```diff
@@ -66,6 +66,134 @@
           "code": "java.class.removed",
           "old": "class io.camunda.zeebe.client.protocol.rest.UserTaskUpdateRequestChangeset",
           "justification": "The classes/methods are only internal and are not user-facing, and as such does not need to maintain any backwards compatibility guarantees."
+        },
```

Comment:
I think this actually needs to go into an `ignored-changes.json` file instead. This one is for persistent changes over multiple releases. In that sense, some of the configurations above shouldn't be in here, I might have actually introduced them in the past 😅

Reply:
I'll move them.